test/e2e/lib/flows/login-flow.js

```diff
@@ -60,18 +60,20 @@ export default class LoginFlow {
 	async login( { emailSSO = false, jetpackSSO = false, useFreshLogin = false } = {} ) {
 		await driverManager.ensureNotLoggedIn( this.driver );
 
-		if (
-			! useFreshLogin &&
-			( await loginCookieHelper.useLoginCookies( this.driver, this.account.username ) )
-		) {
-			console.log( 'Reusing login cookie for ' + this.account.username );
-			await this.driver.navigate().refresh();
-			const continueSelector = By.css( 'div.continue-as-user a' );
-			if ( await driverHelper.isElementPresent( this.driver, continueSelector ) ) {
-				await driverHelper.clickWhenClickable( this.driver, continueSelector );
-			}
-			return;
-		}
+		// Disabling re-use of cookies as latest versions of Chrome don't currently support it.
```

Comment:
Latest versions of Chrome are not working well with cookie re-use, mostly because of the SameSite functionality. I tried a few workarounds, but couldn't make it work on calypso.live